website/content/api-docs/features/consistency.mdx

```diff
@@ -37,15 +64,140 @@ The three read modes are:
+
+#### Limiting Staleness (Advanced Usage)
+
+~> **Not recommended**: we do not recommend using [`dns_config.max_stale`]
+to limit staleness in most use cases because it will exacerbate situations in which responses
+are stale due to server overload.
+
+If [`dns_config.allow_stale`] is `true` (default), `stale` consistency mode is in use for DNS queries.
+To set an upper limit on how stale a response is allowed to be, specify [`dns_config.max_stale`]
+in the configuration of the Consul agent serving DNS queries.
+If the Consul agent handling the response is behind the leader by more than [`dns_config.max_stale`]
+time, the request will be forwarded to the leader for handling like in `default` consistency mode.
+
+The most likely cause for response staleness to exceed [`dns_config.max_stale`] is that
+Consul servers are overloaded by their request load. Under these circumstances, [`dns_config.max_stale`]
+will exacerbate the problem by converting cheap `stale` reads into more expensive `default` reads.
+
+### Consul HTTP API Queries: Defaults and Overriding
+
+All HTTP API read requests use the `default` consistency mode by default
+unless overridden on a per-request basis. Though this default can be changed,
+doing so is not recommended.
+
+Write requests do not support defining a consistency mode.
+All writes follow the behavior of the `consistent` consistency mode.
+
+#### Overridding a Request's Consistency Mode
+
+The HTTP API documentation for each endpoint lists which consistency modes can be used
+to override the default on a per-request basis. To use a supported consistency mode
+other than the default, including the desired consistency mode as a query parameter in the query:
+- `stale`: use the `stale` query parameter
+- `consistent`: use the `consistent` query parameter
+- `default`: use the `leader` query parameter;
+   only relevant [if the default consistency mode is changed to `stale`](#changing-the-default-consistency-mode-advanced-usage)
+
+#### Changing the Default Consistency Mode (Advanced Usage)
+
+~> **Not recommended**: we do not recommend using
+[`discovery_max_stale`](/docs/agent/options#discovery_max_stale)
+to limit staleness in most use cases because it will exacerbate situations in which responses
+are stale due to server overload.
+
+Only the service discovery HTTP API endpoints
+([catalog](/api-docs/catalog), [health](/api-docs/health), and [query](/api-docs/query))
+allow changing the consistency mode used by default. By setting
+[`discovery_max_stale`](/docs/agent/options#discovery_max_stale)
+to a value greater than zero in the configuration of the Consul agent receiving requests,
+the `stale` consistency mode will be used by default unless the data is sufficiently stale.
+Setting a positive value for [`discovery_max_stale`](/docs/agent/options#discovery_max_stale)
+is analogous to [limiting the staleness of Consul DNS queries](#limiting-staleness-advanced-usage)
+by setting [`dns_config.max_stale`].
```

Comment:
> Only the following service discovery HTTP API endpoints support changing the default consistency mode:

That makes it sound like you can only use the `?stale` query param on these endpoints which is not true.

What this paragraph is trying to say is that, for these particular "service discovery" endpoints, it's possible to configure an agent to use `stale` consistency mode _even if not requested by the client_. Which is useful for spreading read load without changing lots of clients to add `?stale` to their requests.

I prefer the bullet points in your suggestion in general but I think overall this section is not as clear as it could be that we are changing the mode the agent serves _without_ any explicit mode in the query param?

Reply:
Ah, ok. Then maybe:

You can configure agents to respond in `stale` consistency mode, even if the client makes requests in a different mode by adding the `?stale` query parameter to one of the following endpoints, depending on the data you want query: 

Is that accurate?

Reply:
Almost:

> You can configure agents to respond in stale consistency mode, even if the client makes requests with no explicit consistency mode in the query string to one of the following endpoints:

It also probably makes sense to explain _why_ you'd want this somewhere though I don't know if it's better to do it after the sentence above and bullet list. But something like:

> This allows Consul operators to force the majority of service discovery read load to be `stale` and so scale across multiple servers without needing to reconfigure or update every application that makes discovery requests.